### PR TITLE
fix default batch size for aggregate

### PR DESCRIPTION
**Pull request by saghm**
As reported in #232, it turns out our fix to the default batch size a little while back actually broke the default batch size for aggregations; aggregations specify the batch size as an option in the command document rather than a flag in the write protocol message, and apparently the server doesn't interpret "0" as meaning "use the server default batch size here", but instead as "use a batch size of 1, and don't even return anything in the first batch. Fixing this just requires not setting the batch size for aggregations when the user doesn't specify one.

As usual, this PR comes chock full of rustfmt changes. You can ignore all of the changes besides the one I commented on, or you can look at how rustfmt seems to have actually improved to the point where the changes are actually improvements rather than just arbitrary.

### CODE REVIEW

File: src/coll/mod.rs [hunk 1]
```diff
@@ -116,19 +114,23 @@ impl Collection {
                 spec = merge_options(spec, aggregate_options);
             }
             None => {
-                let cursor = doc! { "batchSize" => (::cursor::DEFAULT_BATCH_SIZE) };
```
Comment (saghm) [inline src/coll/mod.rs]: This is the only change that matters; the server does not treat an explicit batch size of 0 on aggregation commands as meaning "use the server's default batch size", hence its removal.

**Comment by saghm** [comment]
Random semi-related question: how opposed are you to me just rustfmt'ing the entire repo so we don't get as many of these unrelated changes littered throughout our PRs? I've gotten the impression that it's gotten more stable lately, and it seems to be indenting things in a much more sane way nowadays. I'll still submit it as a PR for you to check out, of course :)

**Comment by kyeah** [comment]
> Random semi-related question: how opposed are you to me just rustfmt'ing the entire repo so we don't get as many of these unrelated changes littered throughout our PRs? I've gotten the impression that it's gotten more stable lately, and it seems to be indenting things in a much more sane way nowadays. I'll still submit it as a PR for you to check out, of course :)

Cool with that. If we could try to run rustfmt into separate PRs as much as possible (so we don't have to do actual-changes hide and seek) that would be wonderful.

**Comment by saghm** [comment]
Awesome! I'll make a PR now, and I'll try to make a point to make another PR whenever a new rustfmt release comes out.

**Pull request merged**